ls1intum/Artemis
# Bugfix/added a scroll bar to feedback editor

### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de. (Tested it on localhost)
### Motivation and Context
This change solves #1397 by expanding the text area to the size of its content.
### Description

I added the function `textareaAutogrow()` in `ngAfterViewInit()` for expanding the feedback textarea to the fitting size.
### Steps for Testing


Precondition: You have created a text exercise, which is ready for assessment
1. Log in to Artemis
2. Navigate to Course Management
3. Write multiple lines of text in it
4. Save the assessment and go back to the dashboard
5. Reopen the assessment
6. The textbox for feedback should still show all lines of feedback

### Screenshots
After reopening an assesment with use of multiple lines of feedback, it should look like this.
![image](https://user-images.githubusercontent.com/50466283/81595849-6b84ea00-93c3-11ea-8f70-aa3321e34530.png)
